command/app.go

```diff
@@ -120,6 +122,14 @@ var app = &cli.App{
 			stat.InitStat()
 		}
 
+		if endpointURL != "" {
+			if !strings.HasPrefix(endpointURL, "http") {
```

Comment:
What do you think about using `net/url.Parse` then checking  `Url.Scheme`?

Reply:
Oh, I missed that it was moved as is from `s3.go`.
https://github.com/peak/s5cmd/blob/914e701426e31a5819a422a4880bdbe03304f9df/storage/s3.go#L85-L87
Lets keep the existing behaviour then.